### DISCUSSION TITLE
Should I load a local model instead of downloading from __init__.py ( Line #33 function _download() )

### DISCUSSION THREAD

**Discussion by qaixerabbas** [openai/whisper#1004] [Q&A] [answered]
I was reading the code and I am interested to load this model from my local storage instead of downloading it from the URL. I am just wondering will it be a good option to download this model and load it locally? Or should I keep using it as it is.

Will there be any advantage of loading a model locally?

https://github.com/openai/whisper/blob/7858aa9c08d98f75575035ecd6481f462d66ca27/whisper/__init__.py#L33

**Comment by glangford** [comment, marked as answer]
> Will there be any advantage of loading a model locally?

`whisper` caches the model for you the first time, so you don't need to do it in other code. You can change the directory it caches model files in, if needed. This is the command line description for changing the directory:

```
--model_dir MODEL_DIR
                        the path to save model files; uses ~/.cache/whisper by default
```
